src/Orckestra.Composer.Cart/Services/Order/OrderHistoryViewService.cs

```diff
@@ -95,7 +95,10 @@ public virtual async Task<OrderHistoryViewModel> GetOrderHistoryViewModelAsync(G
                 shipmentsTrackingInfos = await GetShipmentsTrackingInfoViewModels(orderQueryResult, param).ConfigureAwait(false);
             }
             var orderSettings = await GetOrderSettings(param.Scope).ConfigureAwait(false);
-            
+
+            var orderIds = orderQueryResult.Results.Select(item => Guid.Parse(item.Id)).ToList();
```

Comment:
we can skip this request for PAST orders.
Execute them when  param.OrderTense == OrderTense.CurrentOrders